conf/conf.go

```diff
@@ -36,24 +37,45 @@ var (
 	V = flag.Bool("v", false, "Print version number")
 )
 var (
-	// HomeDir of the config and other files
-	HomeDir string
+	// EditFile borg edit file.
+	EditFile string
+	// ConfigFile borg config file.
+	ConfigFile string
+	// QueryFile borg query file.
+	QueryFile string
 )
 
 func init() {
+	borgDir := borgDir()
+
+	EditFile = filepath.Join(borgDir, "edit")
+	ConfigFile = filepath.Join(borgDir, "config.yml")
+	QueryFile = filepath.Join(borgDir, "query")
+
+	os.Mkdir(borgDir, os.ModePerm)
+	os.Create(EditFile)
+	if _, err := os.Stat(ConfigFile); os.IsNotExist(err) {
```

Comment:
Got it.
BTW, before code is no check `edit` file, so should I confirm to that?
like,

```go
	for _, f := range []string{EditFile, ConfigFile, QueryFile} {
		if _, err := os.Stat(f); os.IsNotExist(err) {
			os.Create(f)
		}
	}
```

vs

```go
	os.Create(EditFile)
	for _, f := range []string{ConfigFile, QueryFile} {
		if _, err := os.Stat(f); os.IsNotExist(err) {
			os.Create(f)
		}
	}
```